book/src/test-cases/tag-import/fixture.css

```diff
@@ -0,0 +1,11 @@
+.this-will-not-be-included {
```

Comment:
for my own understanding, what does `fixture` mean in this context?

Reply:
Something that only exists to be used for a test. 

Commonly used but definitely unclear...

So ... this is a good point. Going forwards I'll use a more clear word.